internal/services/assets/assets_testnet.go

```diff
@@ -2,9 +2,18 @@ package assets
 
 import "github.com/stellar/stellar-disbursement-platform-backend/internal/data"
 
+// EURCAssetIssuerTestnet is the issuer for the EURC asset for testnet
+const EURCAssetIssuerTestnet = "GB3Q6QDZYTHWT7E5PVS3W7FUT5GVAFC5KSZFFLPU25GO7VTC3NM2ZTVO"
```

Comment:
https://stellar.expert/explorer/testnet/asset/EURC-GB3Q6QDZYTHWT7E5PVS3W7FUT5GVAFC5KSZFFLPU25GO7VTC3NM2ZTVO

Reply:
The official website promotes it here: https://www.circle.com/en/eurc